addons/account/models/account_invoiceable_line.py

```diff
@@ -0,0 +1,117 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _, _lt
+from odoo.tools import format_date
+
+
+class InvoiceableLineMixin(models.AbstractModel):
+    _name = 'account.order.line'
+    _description = 'Enables downpayment support for child records of account.order that can be down payments'
+
+    is_downpayment = fields.Boolean(string="Is a down payment",
+                                    help="Down payments are made when creating account moves from an order."
+                                         " They are not copied when duplicating an order.")
+
+    # To override
+    invoice_lines = fields.One2many('account.move.line')
+    display_type = fields.Selection([
+        ('line_section', "Section"),
+        ('line_note', "Note")], default=False, help="Technical field for UX purpose.")
+    product_id = fields.Many2one('product.product')
+    name = fields.Text(compute='_compute_name')
+    order_id = fields.Many2one('account.order')
```

Comment:
It's still there ; the comment is marked outdated because the model was renamed to account.order.mixin. Did you forget, or is it intended ? If so, why ?